spec/models/upload_spec.rb

```diff
@@ -618,4 +618,38 @@ def enable_secure_media
       expect(Upload.secure_media_url?(url)).to eq(false)
     end
   end
+
+  describe "dominant_color" do
+    let(:white_image) { Fabricate(:image_upload, color: "white") }
+    let(:red_image) { Fabricate(:image_upload, color: "red") }
+    let(:not_an_image) { Fabricate(:upload) }
+
+    it "works" do
```

Comment:
I think `works` is too generic here. Are we able to write a description that better describe how we expect the method to behave.